packages/components/src/templates/quote.stories.ts

```diff
@@ -17,174 +17,103 @@ export default {
+            <p class="sd-paragraph sd-paragraph--size-sm">Chairman of the Board</p>
           </div>
         </div>
-        ${information(false, 'xl')}
-      </div>
-    </div>`
+      </div>`
 };
 
-/**
- * This version, called 'inverted', maintains the flexibility of the quote pattern. It primarily
- * focuses on changing all text within the pattern to white, ensuring readability and adaptability
- * for various branding requirements.
- */
-
 export const Inverted = {
   render: () =>
     html` <div class="flex items-end gap-5 p-5 bg-primary">
       <div class="text-left text-white">
         <q class="block mb-6 text-3xl lg:text-4xl leading-tight"
-          >Lorem ipsum dolor sit amet, consetetur sadipscing elitr.</q
-        >
-        <div class="gap-6 flex items-center">
-          ${img()}
-          <div>
-            <p class="sd-paragraph sd-paragraph--inverted font-bold mb-1">Jane Miller</p>
-            <p class="sd-paragraph sd-paragraph--size-sm sd-paragraph--inverted">Job Title</p>
-          </div>
-        </div>
-        ${information()}
-      </div>
-
-      <div class="text-left text-white">
-        <q class="block mb-4 text-2xl lg:text-3xl leading-tight"
-          >Lorem ipsum dolor sit amet, consetetur sadipscing elitr.</q
+          >The environment is becoming increasingly challenging. Despite all the crises, the opportunities are greater
+          than the risks in the long term – this applies in particular to the growth areas of sustainability and
+          retirement pensions.</q
         >
         <div class="gap-6 flex items-center">
-          ${img()}
-          <div>
-            <p class="sd-paragraph sd-paragraph--inverted font-bold mb-1">Jane Miller</p>
-            <p class="sd-paragraph sd-paragraph--size-sm sd-paragraph--inverted">Job Title</p>
-          </div>
-        </div>
-        ${information(false, '3xl')}
-      </div>
-
-      <div class="text-left text-white">
-        <q class="block mb-4 text-xl leading-tight">Lorem ipsum dolor sit amet, consetetur sadipscing elitr.</q>
-        <div class="gap-4 flex items-center">
-          ${img()}
+          <img
+            class="w-16 h-16 rounded-full overflow-hidden object-cover"
+            src="./placeholders/images/HansJoachimReinke.png"
+            alt="Hans Joachim Reinke"
+          />
           <div>
-            <p class="sd-paragraph sd-paragraph--inverted font-bold mb-1">Jane Miller</p>
-            <p class="sd-paragraph sd-paragraph--size-sm sd-paragraph--inverted">Job Title</p>
+            <p class="sd-paragraph sd-paragraph--inverted font-bold mb-1">Hans Joachim Reinke</p>
+            <p class="sd-paragraph sd-paragraph--size-sm sd-paragraph--inverted">Chairman of the Board</p>
           </div>
         </div>
-        ${information(false, 'xl')}
       </div>
     </div>`
 };
 
-/**
- * The `variant` version of this quote provides additional flexibility by allowing you to toggle
```

Comment:
why did we remove all the information on the quote? as we don't have a component but only the template, I see a need for some further information on how to use it.

Reply:
Hi Karl, for the old "patterns" we agreed that they would be part of the "templates" but because they are not "components" we created a different structure for them.